.circleci/config.yml

```diff
@@ -171,6 +184,97 @@ jobs:
           path: /coverage
           desination: coverage
 
+  regression_test:
+    docker:
+      - image: circleci/python:3.7.4
+    working_directory: ~/repo
+    steps:
+      - run: cd ~; git clone https://github.com/lightstep/lightstep-benchmarks.git repo
+      - run: ./scripts/ci_setup.sh
+      - attach_workspace:
+          at: ~/plugin
+      - run: sudo pip install ~/plugin/*.whl
```

Comment:
You'll wan t to change this so that it only installs the wheel for the targeted python version. Take a look how it's done here: https://github.com/lightstep/lightstep-tracer-cpp/blob/master/ci/do_ci.sh#L156-L175